### DISCUSSION TITLE
Previous button

### DISCUSSION THREAD

**Discussion by FlamingFlowair** [pivpn/pivpn#1390] [Feature requests]
Well, it's more a feature request than a issue but please ass a previous button on most parts.

It is already possible for the CA Values to get back and change it again but it's not possible for the public ip vs dns choice.
I took the wront thing and I needed to cancel and loose al the computing time it took to generate my keys :/

Except that, excellent work ! Easy install !

**Comment by jellemdekker** [comment]
I think the best approach would be to let the user configure all the options first and only if they confirm that those are all correct, should the script start generating keys, DH params etc. I'll look into restructuring the steps of the script over the next couple of days to see if it's as easy as I think (and make a PR if it works out). If anybody is interested in this and I haven't replied back after a few days, notify me in a new comment to draw my attention.
